src/OAEdit.cpp

```diff
@@ -83,8 +83,8 @@ status_t TOAEdit::Archive(BMessage *archive, bool deep) const
 
 void TOAEdit::AttachedToWindow()
 {
-// 中にいるBTextViewにメッセージフィルタをかまして
-// ENTERキー、および矢印キーを監視できるようにする。
+// Bite the message filter on the BTextView inside.
```

Comment:
This needs a better translation.

Reply:
If you know Japanese, then sure, go ahead. I can only go by what the translator tells me.